Gems/HttpRequestor/Code/Source/HttpRequestManager.h

```diff
@@ -54,6 +57,7 @@ namespace HttpRequestor
         AZStd::thread                           m_thread;                           // This is the thread that will be used for all async operations
         static const char*                      s_loggingName;                      // Name to use for log messages etc...
         bool                                    m_ownsAwsNativeInitialization = false; // Whether or not this module initialized the native layer
+        AZStd::chrono::milliseconds             m_lastRoundTripTime; // The last round trip time taken to make the http request and get a response.
```

Comment:
Should this be stored in an atomic var? It seems like it might be possible to query it from a different thread where it's getting set.

Reply:
The time you'd normally query lastRoundTripTime would be from within the callback, which is in the same thread that just set lastRoundTripTime. Hmm, you're right though, I could _technically_ call GetLastRoundTripTime anytime I wanted from the main thread...

Reply:
Update! 
m_lastRoundTripTime now atomic in case someone wants to read it outside of the callback (ie: main thread)